### ISSUE TITLE
Cannot use atmos in anaconda3 running python 2.7

### ISSUE THREAD

**Issue by maysamahmadiDev** [atmos-python/atmos#11]
I am trying to install the 'atmos' package for my anaconda3 spyder. Since atmos only works with certain python versions, I reverted back to python 2.7 by running the following:

conda create -n py27 python=2.7  activate py27

Now with this sorted I proceeded to install the atmos package by running the following:

conda install -c mcgibbon atmos

The package is succesfully installed.
I tried testing it by typing this in the console:

import atmos

But I get the following error:

ModuleNotFoundError: No module named 'atmos'

For some reason it cannot find the module I just installed. Is there something I am doing wrong?

**Comment by mcgibbon** [comment]
That's very odd behavior. Unfortunately I could not replicate the issue.

On my OS X machine I ran:
```
$ conda create -n atmos python=2.7
$ conda activate atmos
$ conda install -c mcgibbon atmos
$ python
```
Then within the python console, I ran:
```python
import atmos
```
The import worked just fine. I have never previously installed `atmos` on this machine.

I would suggest running `which python` in your console and seeing if the correct environment version of python is being used. Often disappearing packages are caused by having python installed in multiple ways, and installing a package on a different python installation than the one you're running.
